src/Win32Util.hpp

```diff
@@ -20,7 +20,9 @@
 
 #include "system.hpp"
 
+#include <chrono>
 #include <string>
+#include <thread>
```

Comment:
Suggestion: Move `#include <chrono>` and `#include <thread>` to the `.cpp` file instead since they are not required by anything in the header file.